automatic/graphviz/legal/LICENSE.txt

```diff
@@ -0,0 +1,53 @@
+AT&T has previously made versions of this software available under the AT&T Source Code Agreement, version 1.2D and earlier. If you received a copy of the software under that license agreement, you may continue to use and distribute the same version of the software subject to the terms and conditions of the license agreement under which the software was received. However, current versions of the software are now licensed on an open source basis only under The Common Public License. 
+Common Public License Version 1.0
```

Comment:
Are you sure you have copied the correct license?
When looking at the license located at the specified licenseUrl, it says `Eclipse Public License` on this line.

Reply:
I took a license from graphviz installation directory. Both look almost the same.

Reply:
true, they are more or less the same.
But since you got the license from the installation directory, the VERIFICATION.txt file is wrong, it needs to mention that.

Reply:
And that is the stuff I put in another file by mistake...

Reply:
Done

Reply:
great thanks, now there is only the other item I mentioned.
Since the website supports https, all the urls in the nuspec file (and in the update.ps1) file should make use of that.

After that, I'll add the chocolatey user and merge the PR.

Reply:
fixed.